lib/plugin/instance.rb

```diff
@@ -122,7 +122,21 @@ def enabled?
     @enabled_site_setting ? SiteSetting.get(@enabled_site_setting) : true
   end
 
-  delegate :name, :name_without_prefix, to: :metadata
+  delegate :name, to: :metadata
+
+  def setting_category
+    return if @enabled_site_setting.blank?
+    SiteSetting.categories[enabled_site_setting]
+  end
+
+  def setting_category_name
+    return if setting_category.blank? || setting_category == "plugins"
+    I18n.t("admin_js.admin.site_settings.categories.#{setting_category}")
+  end
```

Comment:
Looks like we don't call `setting_category`, `setting_category_name` outside `Plugin::Instance` so it's better to make them private.

Reply:
Yep makes sense, will fix :+1: